service.go

```diff
@@ -818,9 +818,17 @@ func handleServiceChannelStatus(dc *downstreamConn, params []string) error {
 	n := 0
 
 	sendNetwork := func(net *network) {
+		var channels []*Channel
 		for _, entry := range net.channels.innerMap {
-			ch := entry.value.(*Channel)
+			channels = append(channels, entry.value.(*Channel))
+		}
+
+		sort.SliceStable(channels, func(i, j int) bool {
+			return strings.ReplaceAll(channels[i].Name, "#", "") <
```

Comment:
What is the purpose of trimming `#`? Shouldn't all channels begin with a `#` prefix, removing the need to trim it?

Reply:
Some servers, like on Libera.chat, use ## in some channel names.  I felt #abc and ##abc should be adjacent after sorting.

Reply:
Okay, that's fine by me.